File: app-mvp-dagger/src/main/java/com/xuie/imaginary/data/source/GankSource.java
package xuk.imaginary.data.source;

import xuk.imaginary.data.BaseBean;
import xuk.imaginary.data.GankBean;

import java.util.List;

import io.reactivex.Observable;

/**
 * @author xuie
 * @date 17-7-5
 */
public interface GankSource {
    Observable<List<BaseBean>> get福利(int page);

    Observable<GankBean> getDay(String date);
}

File: app-mvp-kotlin/src/main/java/xuk/imaginary/data/source/GankSource.kt
package xuk.imaginary.data.source

import io.reactivex.Observable
import xuk.imaginary.data.BaseBean
import xuk.imaginary.data.GankBean

/**
 * Created by xuie on 17-7-5.
 */

interface GankSource {
  fun get福利(page: Int): Observable<List<BaseBean>>

  fun getDay(date: String): Observable<GankBean>
}
